.github/bash/branch/checkBranch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
If we would change the file format bash script to python, then would we need to rename the directory from `bash` to `python`?  :)

Reply:
Yes, it could be a little problem in the future. So I changed the directory name from `bash` to `scripts`. 😄